tests/test_text_cross_encoder.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import numpy as np
+import pytest
+
+from fastembed.rerank.cross_encoder import TextCrossEncoder
+
+CANONICAL_SCORE_VALUES = {
+    "Xenova/ms-marco-MiniLM-L-6-v2": np.array([8.500706, -2.5410123]),
+    "Xenova/ms-marco-MiniLM-L-12-v2": np.array([9.330914, -2.0380228]),
+    "BAAI/bge-reranker-base": np.array([6.1573315, -3.6593962]),
```

Comment:
it's not clear how these values were computed, I tried to reproduce it with https://huggingface.co/cross-encoder/ms-marco-MiniLM-L-6-v2 , however the values were different

Reply:
could you please describe how you obtained these values